DependencyInjection/Configuration.php

```diff
@@ -146,7 +151,12 @@ private function getDbalConnectionsNode(): ArrayNodeDefinition
                 ->scalarNode('shard_manager_class')->end()
                 ->scalarNode('shard_choser')->end()
                 ->scalarNode('shard_choser_service')->end()
-                ->booleanNode('keep_slave')->end()
+                ->booleanNode('keep_slave')
+                    ->setDeprecated(
+                        ...$this->getDeprecationMsg('"keep_slave" option is deprecated since doctrine-bundle 2.2. Use keep_replica option instead.')
```

Comment:
```suggestion
                        ...$this->getDeprecationMsg('The "keep_slave" configuration key is deprecated since doctrine-bundle 2.2. Use the "keep_replica" configuration key instead.')
```